### ISSUE TITLE
Add support for reaching certain pages of the website directly

### ISSUE THREAD

**Issue by arthur-1** [ombre-projects/ombre-website#11]
If someone enters a URL like 'ombre.io/downloads' have it automatically load on the download page instead of the home page.

**Comment by yoosofan** [comment]
Yes, it is important. In this new website, this good suggestion will be implemented. You can see my PR on https://github.com/ombre-projects/ombre-website/pull/14 and my experimental download page on https://yoosofan.github.io/ombre-website/downloads.html
